### DISCUSSION TITLE
How to import and use models from Statsforecast in exp.models()?

### DISCUSSION THREAD

**Discussion by NasreddineD** [pycaret/pycaret#3623] [Q&A] [answered]
Hi,

I would like to know how to import and use models from statsforecast in exp.models()?
And how can I check the available models from statsforecast?

Thank you.

**Comment by ngupta23** [comment, marked as answer]
Currently, only ARIMA is available. You can change this using the engine argument. Please check the docstring for details (for setup as well as create_models

**Reply by NasreddineD** [reply]
Thank you
